src/slic3r/GUI/PrintHostDialogs.cpp

```diff
@@ -38,6 +38,11 @@ static const char *CONFIG_KEY_PATH  = "printhost_path";
 static const char *CONFIG_KEY_GROUP = "printhost_group";
 static const char* CONFIG_KEY_STORAGE = "printhost_storage";
 
+static const char* CONFIG_KEY_UPLOADANDPRINT    = "elegoolink_upload_and_print";
```

Comment:
Since these are specific to ElegooPrintHostSendDialog, can we move them into that class?